### DISCUSSION TITLE
Date entry syntax when making query from client side javascript

### DISCUSSION THREAD

**Discussion by feruz111** [graphql/graphql-js#3653] [Q&A]
I have this property that i am inserting into query (to be more precise inserting it into where filter), but it is not working, what is the syntax for date entries (the data type of property in DB i: `timestamp with time zone, default: now()` )
`const createdAt = new Date(value)
....
{ created_at: { _eq: createdAt } }
....
`

P.S. sorry if it is not right place to answer question

**Comment by saihaj** [comment]
How are you representing dates in GraphQL type system? There is no built-in Date in `graphql` so either you use a scalar or you just say every date is an `Int`
- https://www.graphql-scalars.dev/docs/scalars/date
- https://www.graphql-scalars.dev/docs/scalars/date-time
